src/IssueViz/Models/AnalysisIssueLocationVisualization.cs

```diff
@@ -25,12 +25,14 @@
 
 namespace SonarLint.VisualStudio.IssueVisualization.Models
 {
-    public interface IAnalysisIssueLocationVisualization
+    public interface IAnalysisIssueLocationVisualization : INotifyPropertyChanged
```

Comment:
I had to implement it on the interface so that FileNameLocationListItem could register to it.